app/views/bookings/show.html.erb

```diff
@@ -51,6 +51,12 @@
       </tr>
     </table>
   </div>
+
 <%= link_to "Cancel this booking", cancel_booking_path(@booking), method: :patch,
         data: {confirm: 'You are about to cancel you booking for this pokemon, are you sure?'} %>
+
+<%= link_to "Delete this booking",
```

Comment:
Do we need to destroy a booking? I would think that the status is enough to manage it's life cycle no?